concordia/admin/__init__.py

```diff
@@ -205,6 +212,7 @@ class TopicAdmin(admin.ModelAdmin):
 
 @admin.register(Project)
 class ProjectAdmin(admin.ModelAdmin, CustomListDisplayFieldsMixin):
+    AWS_DEFAULT_REGION = os.getenv("AWS_DEFAULT_REGION", "us-east-1")
```

Comment:
Is this used anywhere? If it is needed, it should probably be fed into the Django settings from the environment, and then from the Django settings into whatever code needs it.

Reply:
I would be a strong +1 for removing this – this is the kind of thing which should be set globally in the environment to avoid very confusing errors.

Reply:
Yes that was not needed, I removed it from the code.